src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,116 @@
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertFormatRequest = "INSERT INTO manufacturers(name, country) values(?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement creatManufacturerStatement =
+                         connection.prepareStatement(insertFormatRequest,
+                                 Statement.RETURN_GENERATED_KEYS)) {
+            creatManufacturerStatement.setString(1, manufacturer.getName());
+            creatManufacturerStatement.setString(2, manufacturer.getCountry());
+            creatManufacturerStatement.executeUpdate();
+            ResultSet generateKeys = creatManufacturerStatement.getGeneratedKeys();
+            if (generateKeys.next()) {
+                Long id = generateKeys.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create manufacturer for DB", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String selectManufacturerByIdQuery =
+                "SELECT * FROM manufacturers WHERE is_deleted = false AND id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement getManufacturerByIdStatement =
+                         connection.prepareStatement(selectManufacturerByIdQuery)) {
+            getManufacturerByIdStatement.setLong(1, id);
+            ResultSet resultSet = getManufacturerByIdStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(getManufacturer(resultSet));
+            }
+            return Optional.empty();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer from DB, id: " + id, e);
+        }
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> manufacturerList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement getAllManufacturesStatement = connection.createStatement()) {
+            ResultSet resultSet = getAllManufacturesStatement
+                    .executeQuery("SELECT * FROM taxi_service_db.manufacturers "
+                            + "WHERE is_deleted = false");
+            while (resultSet.next()) {
+                manufacturerList.add(getManufacturer(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all formats from DB", e);
+        }
+        return manufacturerList;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateManufacturerQuery = "UPDATE manufacturers SET name = ?, country = ? "
+                + "WHERE is_deleted = FALSE AND id = ?;";
```

Comment:
look at previous queries and choose one style 
```suggestion
                + "WHERE is_deleted = false AND id = ?;";
```